drivers/platform/xilinx/irq_extra.h

```diff
@@ -48,6 +48,13 @@
 /*************************** Types Declarations *******************************/
 /******************************************************************************/
 
+enum irq_trig_level {
```

Comment:
Can't we add this to irq.h?

Reply:
I did declare it in the header, but I defined it here. This can be different for each platform, even if it might exist everywhere.